### PR TITLE
Multi select tag fe

### LINKED ISSUE #66: Add tag search system
Add a way for users to search for locations that they are more interested in by allowing the user to select a tag for a search.

### PR DESCRIPTION
**Pull request by andy7809**
This PR:

1. Adds a collection of tags before the search button in the side-bar. When a new tag is clicked, the placeType global var is populated with the corresponding string.
2. Updates the colors of the login / signup buttons.

Feel free to change the tags in the html

### CODE REVIEW

File: where-to-travel/src/main/webapp/script.js [hunk 1]
```diff
@@ -80,6 +81,18 @@ script.async = true;
 
 document.head.appendChild(script);
 
+$('.multi-select-pill').click(function () {
+  let pillText = $(this).text();
+  if(pillText === selectedPillText) {
```
Comment (PriyaVarra) [inline where-to-travel/src/main/webapp/script.js]: Where is selectedPillText defined? Is this by any chance supposed to be replaced with placeType?

File: where-to-travel/src/main/webapp/script.js [hunk 2]
```diff
@@ -80,6 +81,18 @@ script.async = true;
...
+    placeType = '';
```
Comment (PriyaVarra) [inline where-to-travel/src/main/webapp/script.js:96]: Am I understanding correctly that in this case, they've clicked on the already selected pill so now there would be no selected type? Just curious.

**Review by PriyaVarra — COMMENTED** [review]
Looks good! Just some minor questions.

File: where-to-travel/src/main/webapp/script.js (hunk 1 above)
Comment (andy7809) [reply where-to-travel/src/main/webapp/script.js]: Yes it is supposed to be placeType, whoops

File: where-to-travel/src/main/webapp/script.js (hunk 2 above)
Comment (andy7809) [reply where-to-travel/src/main/webapp/script.js:96]: Yes, if they click on the same selected placeType, then it sets the placeType to empty

File: where-to-travel/src/main/webapp/script.js (hunk 1 above)
Comment (andy7809) [reply where-to-travel/src/main/webapp/script.js]: Resolved with c493fdc

**Comment by saileshsimhadri** [comment]
LGTM

File: where-to-travel/src/main/webapp/style.css [hunk 3]
```diff
@@ -103,6 +103,25 @@ html {
...
+  background-color: white;
+  border: 1px solid #049688;
+  border-radius: 25px;
+  color: #049688;
+  cursor: pointer;
+  display: inline-block;
+  font-size: 24px;
+  margin: 5px;
+  padding: 5px 10px 5px 10px;
+  transition: box-shadow .2s cubic-bezier(.4,0,1,1),background-color .2s cubic-bezier(.4,0,.2,1),color .2s cubic-bezier(.4,0,.2,1)
```
Comment (wjiamin) [inline where-to-travel/src/main/webapp/style.css:116]: Missing ; for transition?

**Pull request merged**